### DISCUSSION TITLE
Settings/Header Menu - Display Issue

### DISCUSSION THREAD

**Discussion by Arkane** [anapnoe/stable-diffusion-webui-ux#53] [Q&A] [answered]
Hey friends,

So i'm having some issues with the settings display at the top right.
It isn't showing anything when I click on it so i'm unable to change the way the header titles are displayed as well as some other settings i'd like to adjust. Stuff like txt2image, img2image etc...

I installed the fork by using: git clone https://github.com/anapnoe/stable-diffusion-webui-ux.git
This was suggested by another user in a different discussion thread.

Anyone know how to fix this?

![Screenshot 2023-04-13 090655](https://user-images.githubusercontent.com/1338168/231680778-174f2cae-87c2-47d8-b700-21f5e6491b31.png)

**Comment by anapnoe** [comment, marked as answer]
you need to add your own quicksettings there it is empty by default
go to settings tab and click on the menu+ icon  on the right for each setting that you want to be added to the quicksettings aside view

this is the list I am using in the screenshots you can copy paste it to Settings -> User interface ->Quicksettings list
`sd_vae,ui_hidden_tabs,ui_header_tabs,ui_views_order,ui_output_image_fit,ui_show_range_ticks,ui_dispatch_input_release,live_preview_image_fit,extra_networks_default_visibility,extra_networks_cards_size,extra_networks_cards_visible_rows,extra_networks_aside,ignore_overrides,sd_max_resolution,show_progress_every_n_steps`

**Reply by Arkane** [reply]
Thanks!

This was it, worked perfectly ~
